ChaosMod/Effects/db/Player/PlayerBees.cpp

```diff
@@ -0,0 +1,39 @@
+#include <stdafx.h>
+
+// Effect by kolyaventuri
+
+static int CHANCE = 100; // Higher is less chance
+int match = g_random.GetRandomInt(0, CHANCE); // "Seed" for chance to match against
+int particleId;
+
+static void OnStart() {
+    REQUEST_NAMED_PTFX_ASSET("core");
+    while (!HAS_NAMED_PTFX_ASSET_LOADED("core"))
+    {
+        WAIT(0);
+    }
+
+    Ped player = PLAYER_PED_ID();
+    USE_PARTICLE_FX_ASSET("core");
+    
+    particleId = START_PARTICLE_FX_LOOPED_ON_ENTITY("ent_amb_fly_swarm", player, 0, 0, 0, 0, 0, 0, 1.1, false, false, false);
+}
+
+// (kolyaventuri) There's gotta be a better way of doing this...
+static void OnTick() {
+    Ped player = PLAYER_PED_ID();
+    int rand_int = g_random.GetRandomInt(0, CHANCE);
+
+    if (rand_int == match) {
+        APPLY_DAMAGE_TO_PED(player, 1, false, false);
+        // TOOD: Add a damage sound here
```

Comment:
If anyone's aware of a native I can use here to trigger the damage sound, or the IDs of those sounds, that would be great. Or a better native to use :) 

Thinking, would want Michael / Trevor / Franklin's respective damage sounds. The groans / etc. that you hear when they get hit